headers/types/ground_mode/ground_mode.h

```diff
@@ -0,0 +1,19 @@
+// Types used during the ground part (overlay 11)
+
+#ifndef HEADERS_TYPES_GROUND_MODE_H_
+#define HEADERS_TYPES_GROUND_MODE_H_
+
+#include "partner_following.h";
+
+// A global structure holding various pointer to important structure
+struct main_ground_data {
+    undefined* script; //0x0: pointer to script structure
+    undefined* partner_follow_data; //0x4: pointer to the data related to the partner following the player
+    undefined[]* actors; //0x8: pointer to the actors
```

Comment:
I don't think there's much point in making this an array pointer `[]*`. Let's just make it a normal pointer.